mix.exs

```diff
@@ -0,0 +1,22 @@
+defmodule OpenTelemetry.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :opentelemetry_api,
+      version: "0.1.0",
+      elixir: "~> 1.9",
+      start_permanent: Mix.env() == :prod,
+      # We should never have dependencies
+      deps: [],
+      elixirc_paths: ["lib", "src"]
```

Comment:
Why `src` in `elixirc_paths`?

Reply:
It needs to compile the Erlang code as well.

Reply:
But `elixirc_paths` is only about Elixir code, for Erlang there is `erlc_paths`. And these are set to exactly the same values as you are trying to do. So removing that line will change nothing.

Reply:
Ah, ok, never knew.

Reply:
I also did not know this.